### DISCUSSION TITLE
Map at the date line

### DISCUSSION THREAD

**Discussion by cpesch** [mapsforge/mapsforge#1361] [Ideas]
A user pointed me to an interesting behaviour in RouteConverter regarding the date line:

- RouteConverter using mapsforge stops drawing at the date line
- The OSM map continues rendering

Please check the first two pictures on https://forum.routeconverter.com/thread-3652.html for more details

I had a look in the source code but I have no idea how to achieve the same behaviour. Could you give me a hint?

**Comment by devemux86** [comment]
This is not available in Mapsforge.

It is available in VTM OpenGL map library: https://github.com/mapsforge/vtm
